core/src/main/java/com/linecorp/armeria/server/ParameterizedPathMapping.java

```diff
@@ -44,6 +45,8 @@
 
     private static final Pattern VALID_PATTERN = Pattern.compile("(/[^/{}:]+|/:[^/{}]+|/\\{[^/{}]+})+/?");
 
+    private static final Pattern CAPTURE_THE_REST_PATTERN = Pattern.compile("/\\{\\*[^/{}]*}");
```

Comment:
nit: `MULTIPLE_SEGMENTS_MATCHING_PATTERN`?
https://jira.spring.io/browse/SPR-14544?redirect=false

Reply:
I think multiple segments seem to indicate `**`  e.g.  `/service/**`.  🤔
So it's different with this pattern `{*foo}`.

Reply:
That makes sense. 👍 Let's keep it as it is. 😄